book/eha.qmd

```diff
@@ -81,17 +96,75 @@ $$\delta_{ie} = \II(Y_i \leq C_i \wedge E_i = e) \in \{0, 1\},\ e=1,\ldots,k$$
 With this definition, $k$ data sets $(t_i, \delta_{ie}, \bx_i)$, one for each of the $k$ events, can be fit to a separate model in order to obtain cause-specific hazards, as if in the single-event setting.
 Once all hazards are estimated, the all-cause survival probability and CIFs can be calculated. -->
 
-Following @beyersmann.competing.2012, this book assumes that the cause-specific hazards model fully describes the data generating processes and doesn't require an independence assumption for the competing events.
+<!-- Following @beyersmann.competing.2012, this book assumes that the cause-specific hazards model fully describes the data generating processes and doesn't require an independence assumption for the competing events. -->
 
 
 
 ## Multi-state Models {#sec-multi-state}
 
-{{< include _wip.qmd >}}
+Multi-state models can be considered the most general type of survival data, as other types (single-event, competing risks, recurrent events) can be viewed as special cases.
+
+A common multi-state model, the illness-death model, is illustrated in @fig-multi-state, left panel.
+Here healthy subjects can either transition to the terminal evant death (state 2) either directly (transition $0\rightarrow 1$) or after previous illness (transitions $0\rightarrow 1 \rightarrow 2$), without the possibility of back-transitions.
+This is for example appropriate for chronic diseases or terminal illnesses.
+Similar to competing risks, individual transitions are characterised by transition-specific hazards, denoted by $h_{\ell\rightarrow e}(\tau)$ or short $h_{\ell e}(\tau)$, where $\ell$ is the starting state and $e$ the end state  for a transition, $\ell, e \in \{0,\ldots,k\}$.
+Remaining in the same state (transition into the same sate $h_{\ell\ell}(\tau)$) is equivalent to being censored for any transition possible from that state.
+
+![Left: The illness-death model where subjects can transition from state healthy (0) to state diseased (1) and terminal state death (2) without back-transitions. Right: Illustration of a general multi-state model with back-transitions. Here subjects can transition from state healthy (0) to different disease progressions with (partial) recovery and terminal state death.](Figures/survival/multi-state-examples-w-transitions.png){#fig-multi-state fig-alt="Schematic illustration of different multi-state models."}
+
```

Comment:
I don't think these should be combined into a single figure. Naturally I try to directly compare both sides whereas the left side is a specific example you're talking through and right is a general representation